src/pages/steps/LocationStep.tsx

```diff
@@ -81,10 +81,23 @@ const useEditLocation = ({ scope, offerId }) => {
       const { publicRuntimeConfig } = getConfig();
 
       if (!location.country) {
+        await changeAttendanceMode.mutateAsync({
+          eventId: offerId,
+          attendanceMode: AttendanceMode.OFFLINE,
+          location: `${publicRuntimeConfig.apiUrl}/place/${publicRuntimeConfig.cultuurKuurLocationId}`,
+        });
+
         await changeLocationMutation.mutateAsync({
           locationId: publicRuntimeConfig.cultuurKuurLocationId,
           eventId: offerId,
         });
```

Comment:
Is this one still needed? Since `location` is passed to changeAttendanceMode aswell

Reply:
Yes i think it's necessary